layer_v3/sar/template.txt

```diff
@@ -0,0 +1,38 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: <SAR_APP_NAME>
+    Description: "AWS Lambda Layer for Powertools for AWS Lambda V3 [<ARCHITECTURE>] with python <PYTHON_RUNTIME>"
+    Author: AWS
+    SpdxLicenseId: Apache-2.0
```

Comment:
note: historically, SAR didn't have MIT-0 (MIT with no attribution) so @am29d left it as Apache 2.0 back in the days.